### PR TITLE
Added Manager button to Koikatsu Launcher.

### PR DESCRIPTION
**Pull request by drcharlie92**
image

### CODE REVIEW

**Review by ManlyMarco — APPROVED** [review]
SeemsGood

Edit: It looks like you don't check if manager exists, you should hide the button if it doesn't exist like it's done with the update button.

**Comment by drcharlie92** [comment]
Fixed missing files.

File: src/SharedCode/EnvironmentHelper.cs [hunk 1]
```diff
@@ -698,6 +698,27 @@ public static bool StartUpdate()
...
+            {
+                var gameRoot = Path.GetFullPath(GameRootDirectory).TrimEnd('\\', '/', ' ');
+                var updaterPath = Path.Combine(_kkmanagerDirectory, "KKManager.exe");
+
+                if (!File.Exists(updaterPath))
+                    throw new FileNotFoundException("Coult not find the KKManager", updaterPath);
+
+                var args = $"\"{gameRoot}\" {_updateSourcesOverride}";
+
+                return StartProcess(new ProcessStartInfo(updaterPath) { WorkingDirectory = $@"{_kkmanagerDirectory}", Arguments = args }) != null;
```
Comment (ManlyMarco) [inline src/SharedCode/EnvironmentHelper.cs:713]:
```cs
WorkingDirectory = $@"{_kkmanagerDirectory}"
```
could just be
```cs
WorkingDirectory = _kkmanagerDirectory
```

Comment (ManlyMarco) [reply src/SharedCode/EnvironmentHelper.cs]: Fixed it

Comment (drcharlie92) [reply src/SharedCode/EnvironmentHelper.cs]: Same can be done on line 692 for updater method.

**Review by ScrewThisNoise — APPROVED** [review]

**Pull request merged**